src/shared-components/accordion/style.ts

```diff
@@ -21,33 +16,35 @@ export const ExpansionWrapper = styled.div<{ contentHeight: string }>`
   transition: max-height ${ANIMATION.defaultTiming} ease-in-out;
 `;
 
-const getBorderStyle = (isOpen: boolean) => css`
-  border: ${border};
+const getBorderStyle = (theme: ThemeType, isOpen: boolean) => css`
+  border: 1px solid ${theme.COLORS.border};
 
   ${ExpansionWrapper} {
-    ${isOpen && `border-top: ${border};`};
+    ${isOpen && `border-top: 1px solid ${theme.COLORS.border};`};
   }
 `;
 
 export const AccordionBox = styled.div<{
   noBorder: boolean;
   isOpen: boolean;
   disabled: boolean;
+  theme?: ThemeType;
```

Comment:
Emotion 10 recommends exporting a typed `styled` file to allow us to pass the Theme into `styled.div` implicitly: https://emotion.sh/docs/typescript

However, this breaks our `jest` tests when wrapped with a `ThemeProvider`. Emotion 11 will introduce more configuration to handle this more gracefully ([docs](https://deploy-preview-1600--emotion.netlify.app/docs/@emotion/babel-plugin#importmap), [issue comment](https://github.com/emotion-js/emotion/issues/1305#issuecomment-660448554)) but has not yet been released. 

Apparently [Emotion 11 is supposed to be stable](https://github.com/emotion-js/emotion/issues/1305#issuecomment-684979077), though, despite no release, so that is an option to consider if the verbosity is annoying.